### ISSUE TITLE
Using non latin charachters in attribute value

### ISSUE THREAD

**Issue by samanmohamadi** [Sylius/Sylius#8111]
After #7846 It is not possible to use non Latin characters in attribute value.

**Comment by psihius** [comment]
I'd also add that attribute values, especially for "select" type attribute, are used as form names, and that creates issues not just here, but any dynamic forms. I had to override quite a few forms now and give them custom names so it does not error out complaining that there are invalid characters.

**Comment by stefandoorn** [comment]
Running into the same issue @psihius is facing due to non-valid characters used in the form:

```
Uncaught PHP Exception Symfony\Component\Form\Exception\InvalidArgumentException: "The name "value_with_(non-allowed-characters)" contains illegal characters. Names should start with a letter, digit or underscore and only contain letters, digits, numbers, underscores ("_"), hyphens ("-") and colons (":")." at /app/vendor/symfony/symfony/src/Symfony/Component/Form/FormConfigBuilder.php line 865
```

**Comment by stefandoorn** [comment]
PR to fix part of it (select attribute type) in #8134.

**Comment by samanmohamadi** [comment]
It seems that The issue still exist.
 [This function](https://github.com/Sylius/Sylius/pull/8158/commits/2b3fa96dc65c1e7b0cca73bd8e801b15301c99a5#diff-07ca66fd2356d97b982d4c11dad5f539R72) will return empty string for non-latin characters.
